ui/views/components/head.ejs

```diff
@@ -0,0 +1,72 @@
+<meta charset="utf-8" />
+<meta http-equiv="X-XSS-Protection" content="1;mode=block" />
+<meta http-equiv="X-Content-Type-Options" content="nosniff" />
+<meta http-equiv="Permissions-Policy" content="interest-cohort=()" />
+<meta http-equiv="Referrer-Policy" content="strict-origin-when-cross-origin" />
```

Comment:
We should add a CSP meta header in here *now*

Reply:
I removed it since it stopped JS from being loaded

```
bundle.js:1 Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

    at Object.5091 (bundle.js:1:104)
    at __webpack_require__ (bundle.js:1:1049)
    at bundle.js:1:4506
5091 @ bundle.js:1
__webpack_require__ @ bundle.js:1
(anonymous) @ bundle.js:1
```

Reply:
Ah. Well of course. The quickest solution would be to use an environment variable and change the CSP based on dev v prod.

Reply:
Something like `"default-src 'self'; img-src 'self' https:; object-src 'none'; script-src 'self'";` is probably good enough for production. There's a bug in Firefox where you can't `default-src: 'none'` and do SVG symbols.

Reply:
I reckon this might be overkill for the MVP